include/zarray/zarray_impl.hpp

```diff
@@ -17,9 +17,85 @@
 #include <xtensor/xnoalias.hpp>
 #include <xtensor/xscalar.hpp>
 #include <xtensor/xshape.hpp>
+#include <xtensor/xshape.hpp>
+#include "xtensor/xstrided_view.hpp"
 
 namespace xt
 {
+    template <class CTE>
+    class zarray_wrapper;
+
+    template <class CTE>
+    class zchunked_wrapper;
+
+    template <class CTE>
+    class zexpression_wrapper;
+
+    namespace detail
+    {
+        template <class E>
+        struct is_a_strided_view : std::false_type
```

Comment:
Nitpicking: could be `is_xstrided_view` to be consistent with `is_xarray` just below